osquery/distributed/distributed.cpp

```diff
@@ -183,6 +166,21 @@ Status Distributed::acceptWork(const std::string& work) {
               1,
               "Distributed discovery query does not have complete attributes");
         }
+        if (doesQueryRequireSignature(query)) {
+          if (signatures.count(node.first + "_disc") > 0) {
```

Comment:
Is `_disc` something people will use? The abbreviation is odd, why not use the whole word?